tokenizers/src/manager.rs

```diff
@@ -65,12 +67,36 @@ pub enum SearchTokenizer {
     ICUTokenizer,
 }
 
+pub fn language_to_str(lang: &Language) -> &str {
+    match lang {
+        Language::Arabic => "Arabic",
+        Language::Danish => "Danish",
+        Language::Dutch => "Dutch",
+        Language::English => "English",
+        Language::Finnish => "Finnish",
+        Language::French => "French",
+        Language::German => "German",
+        Language::Greek => "Greek",
+        Language::Hungarian => "Hungarian",
+        Language::Italian => "Italian",
+        Language::Norwegian => "Norwegian",
+        Language::Portuguese => "Portuguese",
+        Language::Romanian => "Romanian",
+        Language::Russian => "Russian",
+        Language::Spanish => "Spanish",
+        Language::Swedish => "Swedish",
+        Language::Tamil => "Tamil",
+        Language::Turkish => "Turkish",
+    }
+}
+
 impl SearchTokenizer {
     pub fn name(&self) -> String {
         match self {
             SearchTokenizer::Default => "default".into(),
             SearchTokenizer::Raw => "raw".into(),
             SearchTokenizer::EnStem => "en_stem".into(),
+            SearchTokenizer::Stem { language } => format!("stem_{}", language_to_str(language)),
```

Comment:
@kysshsy We should add documentation for this tokenizer you're creating. The place to add it would be here: https://github.com/paradedb/paradedb/blob/dev/docs/search/full-text/index.mdx#tokenizers